src/content/docs/blog/biome-v2.mdx

```diff
@@ -0,0 +1,269 @@
+
+You can enable the HTML formatter by adding the following to your config file:
+
+```json
+{
+  "html": {
+    "formatter": {
+      "enabled": true
+    }
+  }
+}
+```
+
+## Shout-outs
+
+And now, let's give credits where are due!
+
+<div style={{ display: "flex", justifyContent: "center"}}>
+  <Image
+    src="https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExdjQ2b3d6MjYzMTdsazdzcm41NmM1ZTMzaGcyM2xyeHo2N2k5NmxscyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/QW5nKIoebG8y4/giphy.gif"
+    width="400"
+    alt="it's"
+    height="300"
+  />
+</div>
+
+Special thanks to Vercel, our platinum sponsor, [for sponsoring the type inference work](/blog/vercel-partners-biome-type-inference), led by <Avatar handle="arendjr" />.
+<div style={{ display: "flex", justifyContent: "center"}}>
+  <a href="https://vercel.com" rel="nofollow" title="Vercel" target="_blank">
+    <Image
+      alt="Vercel"
+      src={darkVercel}
+      data-canonical-src={darkVercel.src}
+      width="500"
+      class="dark:sl-hidden"
+      loading="eager"
+    />
+    <Image
+      alt="Vercel"
+      src={lightVercel}
+      data-canonical-src={lightVercel.src}
+      width="500"
+      class="light:sl-hidden"
+      loading="eager"
+    />
+  </a>
+</div>
+
+Special thanks to Depot, our gold sponsor, for providing the project with powerful and fast CI runners. A huge **time saver** and **reliability** booster!
+<div style={{ display: "flex", justifyContent: "center"}}>
+  <a href="https://depot.dev" rel="nofollow" title="Depot" target="_blank">
+    <Image
+      alt="Depot"
+      src={lightDepot}
+      data-canonical-src={lightDepot.src}
+      width="500"
+      class="dark:sl-hidden"
+      loading="eager"
+    />
+    <Image
+      alt="Depot"
+      src={darkDepot}
+      data-canonical-src={darkDepot.src}
+      width="500"
+      class="light:sl-hidden"
+      loading="eager"
+    />
+  </a>
+</div>
+
+Congratulations to <Avatar handle="siketyan" />, who recently became a Core Contributor of the project! Thanks to their contributions,
+the [JetBrains extension](https://plugins.jetbrains.com/plugin/22761-biome) is now stable and supports multiple workspaces.
+
+Thanks to <Avatar handle="conaclos" /> for their massive work in implementing many features such as the Import Organizer revamping,
+the new glob engine, many new linting rules.
+
+Thanks to <Avatar handle="arendjr" />, author of the multi-file engine (scanner and file watcher), the continuous work on the type inference, and complex internal refactors to make Biome more maintainable.
```

Comment:
I understand you're the author of the post, so it may feel weird to include yourself in the shout-outs. I guess I wouldn't include myself either. Just wanted to mention that you absolutely deserve a shout-out for your continuous and tireless efforts in all aspects of the project! :heart: